### DISCUSSION TITLE
欢迎一起讨论联想拯救者Lenovo_R720黑苹果相关

### DISCUSSION THREAD

**Discussion by yulianjie** [yulianjie/Hackintosh_Of_Lenovo_R720#5] [General]
欢迎一起讨论联想拯救者Lenovo_R720黑苹果相关

**Comment by Jesselee0322** [comment]
7700HQ前来汇报，目前完美使用你的EFI没问题，非常感谢

**Reply by yulianjie** [reply]
> 7700HQ前来汇报，目前完美使用你的EFI没问题，非常感谢

感谢你的反馈，毕竟没测试过7700HQ

**Reply by Jesselee0322** [reply]
升级了11.2结果进不去系统了，就重装了11.1，然后发现声音有时候有、有时候没有，几率出现，不知道怎么回事，哈哈

**Reply by yulianjie** [reply]
> 升级了11.2结果进不去系统了，就重装了11.1，然后发现声音有时候有、有时候没有，几率出现，不知道怎么回事，哈哈

我也有这个问题，不过出现的不多，不过一般是启动了windows后重启打开macOS造成的，这时侯在macOS中重启一下就好了

**Reply by Jesselee0322** [reply]
嗯嗯好，感觉笔记本还是比台式机问题多一点，我日常基本都是台式黑果，问题几乎没有
